misk/src/test/kotlin/misk/web/interceptors/RequestLoggingInterceptorTest.kt

```diff
@@ -143,7 +147,9 @@ internal class RequestLoggingInterceptorTest {
       .isEqualTo(500)
     val messages = logCollector.takeMessages(RequestLoggingInterceptor::class)
     assertThat(messages).containsExactly(
-      "ExceptionThrowingRequestLoggingAction principal=caller time=100.0 ms failed request=[fail]"
+      "ExceptionThrowingRequestLoggingAction principal=caller time=100.0 ms failed " +
+        "request=[fail] " +
+        "requestHeaders={accept-encoding=[gzip], connection=[keep-alive]}"
```

Comment:
What's the motivation for adding header dumping to logs? Not sure what the upside is in most cases? Also would lean towards it being opt-in given the noise.

Reply:
Maybe worth adding a boolean config flag to `ActionLoggingConfig`?

Reply:
This does add to a log entry, but it does make the action log longer. I don’t think it makes request logging worse to parse, and the list of headers is heavily filtered already. If it gets bad cost-wise in our logging systems I'd be happy to revert or gate.